bazel/linux/templates/init.template.sh

```diff
@@ -10,4 +10,17 @@ dir="${path%%$relpath}"
 test "$dir" == "$path" || cd "$dir"
 
 set -e
+
+# Remount / with the nosuid flag. Otherwise the following mount fails with:
+# mount: only root can use "--options" option (effective UID is 65534)
+python -c 'import ctypes; exit(ctypes.cdll.LoadLibrary("libc.so.6").mount("", "/", "", 2|32, 0))'
+
+mount --types tmpfs tmpfs /tmp
+
+# Mount the output directory. This directory is shared with the host.
+mkdir /tmp/output_dir
```

Comment:
How does this work with multiple tests running? (I see you have a separate PR for it; not sure if this is answered in a different PR.)
Will two tests running in parallel end up writing in the same dir?

Reply:
The `/tmp/output_dir` is inside each VM and is not shared between them. The host directory where it maps, I believe, is unique.

Reply:
As George says. bazel creates a new temp output directory for each test it runs in parallel. Even if within the VM it has the same name, it is mapped to different directories outside the VM.

Reply:
that makes sense. I was expecting to see here that the mount ties the unique test directory from host to a standard path inside the VM. Is that not available yet?

Reply:
standard path inside the VM is /tmp/output_dir, from the look of the code.